geometry_msgs/Pose2D m_pt
float32 altdepth 			#Altitude for air vehicle, Depth for underwater vehicle, both +ve value
float32 m_pt_radius